<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotterWidget</class>
 <widget class="QWidget" name="PlotterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlotterWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QLineEdit" name="inTitle"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="inDataSource">
     <property name="text">
      <string>Data Source</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>title:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="ycol_QLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>92</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>y-axis:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="xcol_QLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>92</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>x-axis:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QComboBox" name="inXAxis">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>87</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20000</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Plot type</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QComboBox" name="inYAxis">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>87</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>2000</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QComboBox" name="inPlotType">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>line</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>line and marker</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>line and cross</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>marker</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>step-pre</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>step-post</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="maxtstep_QLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Discontinuous plot if time step &gt; days</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QDoubleSpinBox" name="inStep">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLineEdit" name="inWindowSize"/>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Windowing size</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="inXLabel"/>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="inYLabel"/>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="QLineEdit" name="inDataSourceLine"/>
   </item>
   <item row="6" column="4">
    <widget class="QCheckBox" name="inOptRefresh">
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="inPlot">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>97</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>20</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Plot chart</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
